protostar/commands/test/test_case_runners/test_case_runner.py

```diff
@@ -97,3 +101,13 @@ def _map_breaking_reported_exception_to_broken_test_result(
             execution_time=execution_metadata.execution_time,
             captured_stdout=self._output_recorder.get_captures(),
         )
+
+    def _map_skipping_reported_exception_to_skipped_test_result(
```

Comment:
this seems obsolete, since it's only used in 1 place and returns immediately

Reply:
I added it for consistency with other results